ui/packages/app/src/Index.svelte

```diff
@@ -70,7 +70,7 @@
 	export let initial_height: string;
 	export let app_mode: boolean;
 	export let is_embed: boolean;
-	export let theme: "light" | "dark" = "light";
+	export let theme: "light" | "dark" | "system" = "system";
```

Comment:
Could we rename this to `user_theme_variant` or `user_theme_mode` (depending what we decide on for the attribute name) and create a new variable for the actual 'current' theme variant. `active_theme_variant` or `current_theme_variant` or something. I think it is clearer.